nodejs/week1/homework/warmup.js

```diff
@@ -1,35 +1,36 @@
 console.log("inside warmup file");
 
+
 class circle {
     constructor(radius){
         this.raduis=radius
     }
-
-    getDiameter(){
-        
+    getDiameter(){    
       return  console.log(2 * this.raduis);
```

Comment:
Returning the output of `console.log` is not really useful here. For this function, it is expected to return a number.
```suggestion
    const diameter = 2 * this.radius
    console.log(diameter)
     return diameter  ;
```